astrix-bom/pom.xml

```diff
@@ -1,17 +1,62 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>com.avanza.astrix</groupId>
-        <artifactId>astrix-parent</artifactId>
-        <version>1.0.4-SNAPSHOT</version>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>9</version>
+        <relativePath />
     </parent>
```

Comment:
I think the entire `parent` section should be removed. 
The reason for introducing it in https://github.com/netty/netty/pull/6676 was that they copied the parent from `netty-parent` into `netty-bom` instead of `netty-parent`
In the case of astrix, `astrix-parent` does not have a parent, so it should be replaced with no parent in `astrix-bom`

Reply:
Ok, updated in a separate commit now.

Reply:
⭐